budgie-brightness-controller/src/helpers/DimHelper.vala

```diff
@@ -30,11 +36,18 @@ public class DimHelper
 
     private SubprocessHelper subprocessHelper;
     private ConfigHelper configHelper;
+    private ColorSettings? color_settings;
 
     // private int noOfConnectedDev = 0;
 
     public DimHelper()
     {
+        try {
+            color_settings = Bus.get_proxy_sync (BusType.SESSION, "org.gnome.SettingsDaemon.Color",

```

Comment:
Any issues if we remove & add the plugin more than once?  Just wondering if the Bus connection is made twice

Reply:
Tested this. If the applet is removed and readded multiple times, the connection is made again. Likewise if the applet is on the panel multiple times, it connects again. Tested to ensure both instances of the applet were able to correctly see if nightlight mode was on.